resources/create-database-script.sql

```diff
@@ -0,0 +1,24 @@
+create table `Role` (
+	roleID SMALLINT NOT NULL,
+    name varchar(64) NOT NULL,
+    PRIMARY KEY (roleID)
+    );
+
+insert into Role(roleID, name) VALUES (1, 'Admin');
+insert into Role(roleID, name) VALUES (2, 'Employee');
+
+create table `User` (
```

Comment:
With this database table I'd be able to create multiple accounts with the same email and password, and I wouldn't know how to login